OSDOCS-11938: adds greenboot test release note MicroShift (#85918)
openshift/openshift-docs: microshift_release_notes/microshift-4-18-release-notes.adoc

==== Deleting or updating Kustomize manifest resources now documented
With this release, you can delete or upgrade the Kustomize manifest resources. For more information, see xref:../microshift_running_apps/microshift-deleting-resource-manifests.adoc#microshift-deleting-resource-manifests[Deleting or updating Kustomize manifest resources].

[id="microshift-4-18-greenboot-workload-scripts-by-os_{context}"]
==== Greenboot example outputs for image mode for RHEL available

> You can more easily see your system status with an enhancement this release that provides a more detailed explanation and example outputs for checking whether greenboot workload scripts are running properly. See xref:../microshift_running_apps/microshift-greenboot-workload-scripts.adoc#microshift-greenboot-test-workload-health-check-script_microshift-greenboot-workload-scripts[Testing a workload health check script] for details.

Reviewer: ```suggestion
In this release, you can see detailed explanations and example outputs to check whether greenboot workload scripts are running properly. For more information, see xref:../microshift_running_apps/microshift-greenboot-workload-scripts.adoc#microshift-greenboot-test-workload-health-check-script_microshift-greenboot-workload-scripts[Testing a workload health check script].
```
- I tried to tweak the wording a little to focus less on the enhancement and more on what the customer can do with the enhancement.
- OCP doc guidelines prefer leading into a link with `For more information, see`: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences
